### PR TITLE
Simplify readme

### PR DESCRIPTION
**Pull request by akaszynski**
The readme contains a lot of development information. IMO, this is too complex for the average user and it also duplicates much of our web documentation.

Let's simplify this by moving this development information out and into our contributing documentation.

Also updates our runners to use `public-ubuntu-latest-8-cores`.

### CODE REVIEW

**Comment by akaszynski** [comment]
@jorgepiloto, please update the template with this new approach. The readme should be simple enough that a novice user can immediately get started. We also need to focus on the "bottom line up front" rather than providing a slew of development information that will overwhelm the new user.

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -10,45 +10,111 @@ with this guide before attempting to contribute to PyMechanical.
...
+      python -m venv .venv
+
+      # Activate it in a POSIX system
+      source .venv/bin/activate
+
+      # Activate it in Windows CMD environment
+      .venv\Scripts\activate.bat
+
+      # Activate it in Windows Powershell
+      .venv\Scripts\Activate.ps1
```
Comment (MaxJPRey) [inline doc/source/contributing.rst:53]:
```suggestion
.. tab-set::

    .. tab-item:: Windows

        .. tab-set::

            .. tab-item:: CMD

                .. code-block:: text

                    python -m venv .venv
                    .venv\Scripts\activate.bat

            .. tab-item:: PowerShell

                .. code-block:: text

                    python -m venv .venv
                    .venv\Scripts\Activate.ps1


    .. tab-item:: Linux/UNIX

        .. code-block:: text
            python -m venv .venv
            source .venv/bin/activate
  
```

File: doc/source/contributing.rst [hunk 2]
```diff
@@ -74,6 +140,68 @@ when you use the :func:`launch_mechanical() <ansys.mechanical.core.launch_mechan
...
+
+For building documentation, you can run the usual rules provided in the
+`Sphinx`_ ``make`` file. Here is an example:
+
+.. code:: bash
+
+    #  build and view the doc from the POSIX system
+    make -C doc/ html && your_browser_name doc/html/index.html
+
+    # build and view the doc from CMD / PowerShell environment
```
Comment (MaxJPRey) [inline doc/source/contributing.rst:154]:
```suggestion
    # build and view the doc from Windows environment
```

File: doc/source/contributing.rst [hunk 3]
```diff
@@ -74,6 +140,68 @@ when you use the :func:`launch_mechanical() <ansys.mechanical.core.launch_mechan
...
+    .\doc\make.bat html
+    start .\doc\_build\html\index.html
+
+
+However, the recommended way of checking documentation integrity is to use
+``tox``:
+
+.. code:: bash
+
+    tox -e doc && your_browser_name .tox/doc_out/index.html
```
Comment (MaxJPRey) [inline doc/source/contributing.rst:165]:
```suggestion
    tox -e doc && <your_browser_name> .tox/doc_out/index.html
```

**Review by samigithub2022 — APPROVED** [review]

**Pull request merged**